addons/mail/static/src/components/call_invite_request_popup/call_invite_request_popup.scss

```diff
@@ -7,8 +7,7 @@
 }
 
 .o_CallInviteRequestPopup_partnerInfoImage {
-    width: 70%;
-    height: 70%;
+    --avatar: 70%;
```

Comment:
`--avatar` : this is really unfortunate :/
Can we get `--avatar-size` instead? We can avoid using `$grid-breakpoints` if it's a problem

Reply:
The problem is not `$grid-breakpoints` but the API uses the same name as the class.
So, if we want to get `--avatar-size`, the classes will be `avatar-size-[size]`.
https://getbootstrap.com/docs/5.1/utilities/api/#css-variable-utilities

Reply:
> So, if we want to get --avatar-size, the classes will be avatar-size-[size].

Ok for me. I prefer a verbose class rather than a generic CSS variable